flang/lib/Lower/OpenMP/OpenMP.cpp

```diff
@@ -1492,7 +1466,7 @@ genParallelOp(lower::AbstractConverter &converter, lower::SymMap &symTable,
     firOpBuilder.createBlock(&region, /*insertPt=*/{}, allRegionArgTypes,
                              allRegionArgLocs);
 
-    llvm::SmallVector<const semantics::Symbol *> allSymbols = reductionSyms;
+    llvm::SmallVector<const semantics::Symbol *> allSymbols(reductionSyms);
```

Comment:
ultra-nit, feel free to ignore

Personally I don't like using `()` here because it can parse (to humans and computers) like a function declaration. 
```suggestion
    llvm::SmallVector<const semantics::Symbol *> allSymbols{reductionSyms};
```

Reply:
I made the change here and in other places touched by this patch. However, I just noticed this in the coding standards guide: https://llvm.org/docs/CodingStandards.html#do-not-use-braced-initializer-lists-to-call-a-constructor. My understanding is that "function-like" syntax is preferred unless it's some kind of aggregate initialization.

Reply:
Thanks for linking me to this!